traits/tests/test_interface_checker.py

```diff
@@ -392,7 +393,7 @@ class Foo(HasTraits):
         class Bar(HasTraits):
             foo = Instance(IFoo)
 
-        b = Bar(foo=Foo())
+        Bar(foo=Foo())
```

Comment:
im guessing this too is a test where we simply instantiate the object and expect to see/not see errors.

Reply:
Yes, this was a drive-by fix to eliminate a `flake8` warning about an unused variable, without changing the semantics of the test.